api/graphql/schema-types-defs/Common.graphql

```diff
@@ -0,0 +1,33 @@
+type source {
+  sourceId: String
+  gurmukhi: String
+  unicode: String
+  english: String
+  pageNo: Int
+}
+
+type raag {
+  raagId: Int
+  gurmukhi: String
+  unicode: String
+  english: String
+  startAng: Int
+  endAng: Int
+  raagWithPage: String
+}
+
+type writer {
+  writerId: Int
+  gurmukhi: String
+  unicode: String
+  english: String
+}
+
+type transliteration {
+  english: String
+  hindi: String
+  en: String
+  hi: String
```

Comment:
just general question: whats the need for hi and hindi at the same time

Reply:
at the same time, not sure. I think the longer one is suppose to be deprecated.